PracticeNotebook.LINQ/LinqQuery.cs

```diff
@@ -279,16 +278,31 @@ public void GetProfessorCount()
 
         #region Async-query
         // The return type of async method is Task.
+        private async Task<List<Professor>> GetProfessors()
+        {
+            return await Task.FromResult(new List<Professor>
```

Comment:
Do you think it would be better if `return await Task.FromResult(_professors);`? So it doesn't need to instantiate the list repeatedly.